tasks/src/image-to-text/data.ts

```diff
@@ -0,0 +1,58 @@
+import type { TaskDataCustom } from "../Types";
+
+const taskData: TaskDataCustom = {
+	datasets: [
+		{
+			// TODO write proper description
+			description: "Dataset from 12M image-text of Reddit",
+			id:          "red_caps",
+		},
+        {
+			// TODO write proper description
+			description: "Dataset from images generated by Stable diffusion on Discord",
+			id:          "poloclub/diffusiondb",
+		},
+
+	],
+	demo: {
+		inputs: [
+			{
+				filename: "savanna.jpg",
+				type:    "img",
+			},
+		],
+		outputs: [
+			{
+				label:   "Detailed description",
+				content: "a herd of giraffes and zebras grazing in a field",
+				type:    "text",
+			},
+		],
+	},
+	metrics: [
+		{
+			description: "",
+			id:          "",
+		},
+		{
+			description: "",
+			id:          "",
+		}
+	],
+	models: [
+        {
+            description: "Donut consists of a vision encoder (Swin Transformer) and a text decoder (BART)",
```

Comment:
The audience of this page is software developers and beginners, so we try to avoid using terms (like architectures). Can you simplify the description?